README.md

```diff
@@ -30,3 +30,71 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+
+# My Analysis
+
+The base case of this recursive algorithm states that if n is less than or 
+equal to 1, then it will return. In the case that the input is 1 or less, 
+this algorithm has a constant runtime. However, when n is greater than 1, 
+things get more intersting.
+
+Looking at the algorithm, it can be boiled down to 3 recursive calls and a 
+triple-nested for loop. Starting with the nested for loop, the base for 
+loop (with the iterator 'k'), we see that it will complete n\*n times since 
+k will only ever iterate by 1. Similarly, with the second for loop (with 
+the iterator 'j'), we see it will complete n times, and for the last for 
+loop (with the iterator 'i'), it will run n\*n times. Written out, we can 
+write $\Theta(n^2\*n\*n^2)$ or simplify to $\Theta(n^5)$.
+
+We also see that there are 3 seperate non-nested recursive calls where the 
+input n is divided by 3. Written into a recursive relation, we get: 
+
+$$T(n) = 3T(n/3)$$
+
+Adding the triple-nested for loop into the equation:
+
+$$T(n) = 3T(n/3) + \Theta(n^5)$$
```

Comment:
You can't really use $\Theta$ expressions here; this only comes in at the end once you've simplified the entire thing.

Reply:
With the $\Theta$ expression come into play when a generate a pattern where I have the variable k? Or perhaps even later after I plug in for T(1) on line 72?

Reply:
No, only at the end you derive a $\Theta$ expression.